scripts/validate-ci-config.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
```

Comment:
Take a look at the lint-staged section of js-sdk package.json. We probably don't need to run this before every commit. We could just do the linting only when the circle config file changes.